springapp/src/main/java/com/javaacademy/robot/converters/BookConverter.java

```diff
@@ -63,11 +63,26 @@ public Book toEntity(BookDto dto) {
         return entities.stream().map(this::toDto).collect(Collectors.toList());
     }
 
+    List<String> parseAuthors(List<String> authors) {
+        if (authors == null) {
+            return Collections.emptyList();
+        } else if (authors.size() == 1 && authors.get(0).equals("")) {
```

Comment:
magic number, what is more it would be better to create small private method, which would return boolean and as name of this method you could name what is going on here